shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -77,6 +78,22 @@ void Reset() {
 
 #pragma mark - Private interface declaration.
 
+/**
+ * This view is the wrapper of a FlutterView, and it is useful to attach other
+ * native views, e.g. the FlutterTextField, without affecting the accessibility
+ * subtree of FlutterView.
```

Comment:
Maybe:
A wrapper around FlutterView. FlutterViewWrapper is a convenience class that allows attaching AppKit views such as FlutterTextField without affecting the accessibility subtree of the FlutterView itself.